packages/lobster-tool-codebeamer/README.md

```diff
@@ -11,31 +11,50 @@ requirements management tool
 
 ## Tools
 
-* `lobster-codebeamer`: Extrat requirements from codebeamer.
+* `lobster-codebeamer`: Extract requirements from codebeamer.
 
 ## Configuration
-This tool works with an optional config file. In it you can declare which 
-codebeamer fields should be used as 'refs' reference in the codebeamer file.
-
-For the 'refs' reference in config file you can write:
-
-```
-{
-"refs" : "cb-fieldname"
-}
-```
-or
-```
-{
-"refs" : ["cb-fieldname"]
-}
-```
-or
-```
-{
-"refs" : ["cb-fieldname1", "cb-fieldname2"]
-}
-```
+This tool works with an optional config file.
+*   You can declare the  codebeamer fields which shall be used as 'refs' reference in the output file.
+
+    For the `refs` reference in config file you can write:
+
+    ```json
+    {
+      "refs" : "cb-fieldname"
+    }
+    ```
+    or
+    ```json
+    {
+      "refs" : ["cb-fieldname"]
+    }
+    ```
+    or
+    ```json
+    {
+      "refs" : ["cb-fieldname1", "cb-fieldname2"]
+    }
+    ```
+
+* Schema (kind)
+
+  You can also specify the type of schema for the resulting output file. The supported values for the kind field are:
+  1. Activity: Sets the schema to lobster-act-trace.
+  2. Implementation: Sets the schema to lobster-imp-trace.
+  3. Requirement: Sets the schema to lobster-req-trace.
+
+  If the kind is not specified, the tool will default to Requirement, and the schema lobster-req-trace will be used.
+
+  Here’s an example configuration:
+  ```json
+  {
+    "kind": "Activity",  // Specifies schema
```

Comment:
Here, I suppose it should be "schema"
because, If I add "kind" and "refs" to the config file it gives an error message - 

> KeyError: "The provided config keys are not supported! supported keys: 'refs, schema'"